proceeding/text/introduction.tex

```diff
@@ -9,4 +9,18 @@ \section{Introduction}
 \item Because of an increasing interest in the field, there is a need for a hub of VHE data (GeV, TeV) across many different catalogs.
 This is what gamma-sky.net was created for.
 
+\pagebreak
+
+\paragraph{}        The field of high-energy (HE) and very-high-energy (VHE) astronomy is growing tremendously – while only
```

Comment:
I've never used the `\paragraph` and `par`
Can you just remove those?

Usually an empty line in LaTeX means that a new paragraph starts and no further markup commands are needed.

https://en.wikibooks.org/wiki/LaTeX/Paragraph_Formatting